test/test_brt.py

```diff
@@ -11,7 +11,7 @@
 
 @pytest.mark.localdata
 @pytest.mark.slow
-def test_brt_p(mock_nfs_mount):
+def test_brt(mock_nfs_mount):
```

Comment:
ah, ok. I renamed this to allow `-k` option in pytest 
ie:
`This will run tests which contain names that match the given string expression,`
That same substring is used in other test names. Added `_p` to make uniq

Reply:
You can avoid this by running the following command `pytest test/test_brt.py::test_brt`.